packages/evolution-common/src/services/baseObjects/baseObjectsAndCompositionDoc.md

```diff
@@ -1,3 +1,5 @@
+# This documentation is deprecated
```

Comment:
If you get  chance, maybe just point to the new documentation

Reply:
Added